qt_gui_cpp/include/qt_gui_cpp/ros_pluginlib_plugin_provider.h

```diff
@@ -295,8 +295,8 @@ class RosPluginlibPluginProvider
     std::string manifest_path = class_loader_->getPluginManifestPath(lookup_name);
     //qDebug("RosPluginlibPluginProvider::parseManifest() manifest_path \"%s\"", manifest_path.c_str());
     tinyxml2::XMLDocument doc;
-    bool loaded = doc.LoadFile(manifest_path.c_str());
-    if (!loaded || doc.Error())
+    bool error = doc.LoadFile(manifest_path.c_str());
```

Comment:
> The result of tinyxml2::loadFile is 0 on success and otherwise the enum of the error message.

Should the type no be different than `bool`?

Reply:
Sorry, it was a bit lazy. Updated